apps/rush-lib/src/cli/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -95,6 +95,11 @@ export function extractVersionFromPnpmVersionSpecifier(version: string): string
     if (versionParts.length === 5 && isScoped) {
       extractedVersion = versionParts[3]; // e.g. "3.1.1"
     }
+
+    // e.g. "path.pkgs.visualstudio.com/@scope/depame/1.4.0"
+    if (!extractedVersion && versionParts[versionParts.length - 1].match(/^(\d*)\.(\d*)\.(\d*)/)) {
```

Comment:
>versionParts.length [](start = 42, length = 19)

Is it possible for `versionParts.length` to be zero here?  What would versionParts[-1] do then?